runtime/tcore.c

```diff
@@ -0,0 +1,114 @@
+#include "tcore.h"
+
+#include "lua.h"
+#include "lauxlib.h"
+
+#include "lobject.h"
+#include "lstate.h"
+#include "ltm.h"
+
+const char * titan_tag_name(int raw_tag)
+{
+    if (raw_tag == LUA_TNUMINT) {
+        return "integer";
+    } else if (raw_tag == LUA_TNUMFLT) {
+        return "float";
+    } else {
+        return ttypename(novariant(raw_tag));
+    }
+}
+
+void titan_runtime_arity_error(
+    lua_State *L,
+    int expected,
+    int received
+){
```

Comment:
Could we close this `)` at the previous line (if yes fix all functions).

Reply:
The pretty printer prefers the `)` on a line by itself, similar to how `}` works. I would like to be consistent with that.